frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -244,15 +255,33 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
     }
   }, [dispatch, leafletMouseEvent]);
 
+  /**
+   * Only display the edit button if the user has the correct claim and the property is owned by their agency.
+   */
   const ConditionalEditButton = () => (
     <>
       {disabled &&
         (keycloak.hasClaim(Claims.ADMIN_PROPERTIES) ||
-          keycloak.hasClaim(Claims.PROPERTY_EDIT) ||
-          keycloak.agencyId === parcelDetail?.agencyId ||
-          keycloak.agencyId === buildingDetail?.agencyId) && (
-          <EditButton onClick={() => setDisabled(false)} />
-        )}
+          (keycloak.hasClaim(Claims.PROPERTY_EDIT) &&
+            (keycloak.agencyId === parcelDetail?.agencyId ||
+              keycloak.agencyId === buildingDetail?.agencyId) && (
+              <EditButton onClick={() => setDisabled(false)} />
+            )))}
+    </>
+  );
+
+  /**
+   * Only display the delete button if the user has the correct claim and the property is owned by their agency.
+   */
+  const ConditionalDeleteButton = () => (
+    <>
+      {disabled &&
+        (keycloak.hasClaim(Claims.ADMIN_PROPERTIES) ||
```

Comment:
`admin-properties` or a user with `property-edit` and the property belongs to their agency.  This however is still wrong, as a user should be able to view and edit sub-agency properties.  I'll look into this later.